sdk/storage/Azure.Storage.Common/src/Shared/StorageBearerTokenChallengeAuthorizationPolicy.cs

```diff
@@ -82,6 +83,23 @@ private async ValueTask<bool> AuthorizeRequestOnChallengeInternalAsync(HttpMessa
         {
             try
             {
+                if (AuthorizationChallengeParser.IsCaeClaimsChallenge(message.Response))
```

Comment:
Does the CAE path not need to parse the other parameters the policy is looking for in the challenge below? Specificallyh the authorization_uri and resource_id?

Reply:
Isn't that logic only relevant for the tenant discovery? I'm not sure if both a token revocation and a tenant discovery would happen in the same challenge.